storybook/src/Models/SFPMWalletAccountsModel.qml

```diff
@@ -0,0 +1,32 @@
+import QtQuick 2.0
+
+import SortFilterProxyModel 0.2
+import utils 1.0
+
+SortFilterProxyModel {
```

Comment:
Do we really need that proxy in storybook models? Maybe we could have only `WalletAccountsModel` with role `color` defined directly?
That proxy logic is now internal implementation detail of `Wallet/stores/RootStore.qml`. And that details shouldn't permeate to storybook's models.

Reply:
mmmm.. it's just an analogous model. We can add the `color` property inside `WalletAccountsModel` directly if you are more comfortable with that but.. there is this filter by `walletType` that will not be applied!

Reply:
Basically, for components in storybook we don't need that filtering at all.

Reply:
So it reflects what we do in `RootStore` but in fact it's not needed at all in storybook imo.